### PR TITLE
Add Scripts/runcts script as a convenience for running Vulkan CTS tests.

### PR DESCRIPTION
**Pull request by billhollings**
Update `.gitignore` to ignore CTS artifacts in `Scripts` directory.

### CODE REVIEW

File: Scripts/runcts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+         shift 2
+         ;;
+       -p)
+         is_portability="Y"
+         shift 1
+         ;;
+       --portability)
+         is_portability="Y"
+         shift 1
+         ;;
```
Comment (cdavis5e) [inline Scripts/runcts:67]: You could probably consolidate these into one case like this:
```suggestion
       -p|--portability)
         is_portability="Y"
         shift 1
         ;;
```

File: Scripts/runcts [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+if [ "${results_file}" == "" ]; then
+	results_file="${caselist_file}-results.txt"
+fi
+
+
+# -------------- MoltenVK configuration --------------------
+
+if [ "${is_portability}" != "" ]; then
+	export MVK_CONFIG_API_VERSION_TO_ADVERTISE=4194304
```
Comment (cdavis5e) [inline Scripts/runcts:92]: That magic number looks pretty mysterious. You should probably have a comment here explaining it.

**Review by cdavis5e — CHANGES_REQUESTED** [review]
Just a few changes.

File: Scripts/runcts (hunk 1 above)
Comment (billhollings) [reply Scripts/runcts]: Done.

File: Scripts/runcts (hunk 2 above)
Comment (billhollings) [reply Scripts/runcts:93]: Done. 

The values are documented in `vk_mvk_moltenvk.h`...and in the documentation in this script file ...but fair enough...doesn't hurt to explain how the values used here as derived.

**Review by cdavis5e — APPROVED** [review]

**Pull request merged**